src/static/renderer.ts

```diff
@@ -56,11 +57,11 @@ elmApp.ports.requestISocksProxy.subscribe(() => {
 
 const handleWebRequest = (data, respond) => {
   const message = JSON.parse(data);
-  const webRequest = CLIQZ.app.modules['webrequest-pipeline'].background;
+  const webRequest = cliqzApp.modules['webrequest-pipeline'].background;
   const eventName = message.functionName;
   console.log('received: ', message);
 
-  if (!webRequestEvents.includes(eventName) || !(eventName in webRequest)) {
```

Comment:
why not `includes`?

Reply:
It's not supported by typescript by default I think. We need to target `es7`: https://github.com/Microsoft/TypeScript/issues/11890

Reply:
as we run on node8 it should be fine to set that target